docs/internal/investigations/dapr-remote-runtime-configuration/configuration_concept.md

```diff
@@ -0,0 +1,25 @@
+# Passing configuration to runtimes concept
+
+## Overview
+
+This document describes the concept of passing a configuration to runtimes from the central point with the usage of [DAPR](https://dapr.io).
+
+![Concept image](assets/concept.png?raw=true "Concept")
+
+## Idea
+
+The main idea is to have a component in Kyma Control Plane (Runtime Director on image), which would be a point of communication with Runtimes through Agents and provide a configuration for them. The configuration would be for example an URL and credentials to some external service (e.g. logging service). After each runtime fetch the new configuration, the DAPR bindings should be updated and populated to the DAPR sidecars.
```

Comment:
I have an impression from that sentence that the main reason of having the Runtime Director component is to have a possibility to deliver configuration to the runtimes. I would rephrase this to more underline the intention of having the Runtime Director for runtime management. Then in some details we can focus on the configuration part. 

I'm also missing the big picture describing the architecture presented on the concept image.